### PR TITLE
Fix Error http Error and new Docker-compose file

### PR DESCRIPTION
**Pull request by Happyfeet01**
This are my changes to the Repository.

i edit an File to solve an Error when connect over http. Only visible when no entrys exist in Database. See here: https://github.com/gutmensch/docker-dmarc-report/issues/9#issuecomment-744767393

also added an volume to the Compose File to store the Reports on the host File system.

### CODE REVIEW

File: manifest/var/www/viewer/dmarcts-report-viewer-config.php [hunk 1]
```diff
@@ -14,5 +14,6 @@
 
 $default_hostlookup = 1;  # 1=on 0=off (on is old behaviour )
 $default_sort = 1;  # 1=ASCdening 0=DESCending (ASCending is default behaviour )
+$imapignoreerror = 1;
```
Comment (gutmensch) [inline manifest/var/www/viewer/dmarcts-report-viewer-config.php:17]: Uhm, this variable is for the parser and not for the viewer? The viewer is not talking to IMAP AFAIK? In the report parser conf file this variable is described and set to 0 by default (maybe debatable)

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+services:
+  dmarc-report:
+    image: dmarc-report:latest
+    hostname: dmarc-report
+    container_name: dmarc-report
+    restart: always
+    depends_on:
+      - db
+    ports:
+      - "4446:80"
```
Comment (gutmensch) [inline docker-compose.yml:12]: can we make this at least 8080:80 or something like this? I don't think a port like 4446 adds to the example clarity here, even if this is maybe the port you are using, but it should denote that it is a http service (hence I'm suggesting 8080 as alt-http)

**Review by gutmensch — COMMENTED** [review]
Sorry for the late response and thanks for sending in the PR, nice work! Just slight remarks for clarification :)

File: docker-compose.yml (hunk 2 above)
Comment (Happyfeet01) [reply docker-compose.yml:12]: sure

**Pull request closed without merging**